core/src/main/resources/hudson/Messages_es.properties

```diff
@@ -55,3 +55,4 @@ PluginManager.PluginDoesntSupportDynamicLoad.RestartRequired=El plugin {0} no so
 PluginManager.PortNotANumber=El puerto no es un n�mero
 AboutJenkins.Description=Eche un vistazo a la informaci�n sobre la versi�n y la licencia.
 PluginManager.PortNotInRange=El puerto no est� en el rango de {0} hasta {1}
+PluginManager.CheckUpdateServerError=Hubo errores comprobando los servidores de actualizaciones: {0}
```

Comment:
I know it's out the scope of this PR, but what about to complete the translation of the messages file? It would be nice since you have added your message :)

Reply:
I agree about the usefulness, while at it, *but* I'm strongly against adding such commit in this PR. Keep PRs small, file many if needed, we all win :-). Shorter reviews, more chance to get merged and not block oneself for an unrelated change that someone would disagree on.

Reply:
An I agree with you @batmat in general and obviously if it would be a reformat of the code. However, in this case it's just text in his native language and he's modifying the file, that's why I'm suggesting. If he shouldn't have to add his message, my vote would be absolutely for not changing the content of the file.
IMO, if we leave that sort of changes for other PRs, the messages will likely keep out-of-date, I'm afraid :disappointed: 

But if everyone else that review the PR think it's better to open a new PR, it's fine for me.

Reply:
@batmat In most cases I actually agree with you. I just see this file a bit different just because it's only one of the resource boundles for a concrete message, but as I say before, I'm happy if it's done in another PR :)